The hunks are overlapping by more than just context. This would not be a problem
for rapidquilt, but patch rejects such patches, so we do it too.
(There are some special cases when patch accepts such patches, but that is likely
a bug, not intention.)
--- filename
+++ filename
@@ -5,7 +5,7 @@ foo
 aaa
 bbb
 ccc
-ddd
+ddd modified
 eee
 fff
 ggg
@@ -8,7 +8,7 @@ bar
 ddd modified
 eee
 fff
-ggg
+ggg modified
 hhh
 iii
 jjj
